UnityProject/Assets/Scripts/UI/Canister/NetWheel.cs

```diff
@@ -0,0 +1,25 @@
+
+using System;
+using UnityEngine;
+
+public class NetWheel : NetUIElement
+{
```

Comment:
Maybe add a summary comment here? as it isn't might not be fully clear what it's for for everyone.